docs/helpshift-widget/configuration.mdx

```diff
@@ -0,0 +1,296 @@
+
+- Disabling user-initiated screenshots (users will not be able to attach files, including images, using Web Chat).
+- Making sure that Personally Identifiable Information (PII) such as name and email are not collected by Web Chat (using Identity Bot and/or the helpshiftWidgetConfig object). This means that if you set `userName` and `userEmail`, with `fullPrivacy` set to `true`, Helpshift will not use the `userName` and `userEmail` values.
+
+Moreover, in scenarios where the user attaches objectionable content, it becomes a huge COPPA compliance concern. This option helps to solve this problem.
+
+##### Example Embed Code
+
+```javascript
+// The following part remains as it is in the Embed Code.
+var PLATFORM_ID = "<YOUR_PLATFORM_ID>",
+  DOMAIN = "<YOUR_DOMAIN>";
+  APP_ID = "<YOUR_APP_ID>";
+  WIDGET_TYPE = "<WIDGET_TYPE>";
+
+window.helpshiftWidgetConfig = {
+  platformId: PLATFORM_ID,
+  domain: DOMAIN,
+  appId: APP_ID,
+  widgetType: WIDGET_TYPE
+  // other options, if applicable
+  fullPrivacy: true
+};
+```
+
+## Initiating chat {#initiate-chat}
+
+Developers can start a new chat when the previous issue is resolved, without needing any user action like clicking the `New Conversation` button or going through the post resolution flow like feedback bot for the previously resolved issue.
+
+To do the above mentioned action, the developer will have to set `initiateChatOnLoad` config in `window.helpshiftWidgetConfig` and then use the [updateHelpshiftWidgetConfig](https://developers.helpshift.com/helpshift-widget/api/#update-helpshift-widget-config) API to start the new chat when the previous issue is resolved.
+
+##### Example Embed Code
+
+```javascript
+window.helpshiftWidgetConfig.initiateChatOnLoad = true;
+
+Helpshift("updateHelpshiftWidgetConfig");
+```
+
+To open the widget, [Open Widget API](https://developers.helpshift.com/web-chat/api/#open-widget) can be used.
+
+## Initial user message {#initial-user-message}
+
+<InitialUserMessageDescription />
+
+##### Example Embed Code
+
+```javascript
+window.helpshiftWidgetConfig.initialUserMessage = "Some initial user message";
+
+Helpshift("updateHelpshiftWidgetConfig");
+```
+
+## Conversation prefill text {#conversation-prefill-text}
+
+<ConversationPrefillTextDescription />
+
+##### Example embed code:
+
+```javascript
+window.helpshiftWidgetConfig.conversationPrefillText = "INSERT YOUR PREFILL TEXT";
+Helpshift("updateHelpshiftWidgetConfig");
+```
+
+To open the widget, [Open Widget API](https://developers.helpshift.com/web-chat/api/#open-widget) can be used.
+
+## Showing a Particular FAQ Section {#faq-section-view}
+
+You can use this configuration to show FAQs from a particular FAQ section.
+
+**You will need the `publish-id` of the FAQ section to use this API:**
+
+<Image src="/static/books/misc/section-publish-id.png" width="full" />
+
+##### Example embed code:
+
+```javascript
+window.helpshiftWidgetConfig.sectionId = "333";
+Helpshift("updateHelpshiftWidgetConfig");
```

Comment:
This flow looks slightly misleading. `sectionId` should be part of the initial config, and set before Helpshift widget is opened. Here it can be interpreted like you update you config & call Helpshift API to open particular section on the fly.

Reply:
I think, then it needs to be added after the `widgetType` config.

Reply:
Done